File: backend/database/models/highprice.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  const highPrice = sequelize.define('highPrice', {
    price: {
      type: DataTypes.DECIMAL,
      allowNull: false,
      validate: {
        notNull: {
          msg: 'Value can not be null.'
        },
        isDecimal: {
          args: true,
          msg: 'Price must be a decimal.',
        },
      }
    }
  }, {
    freezeTableName: true,
    tableName: 'highPrice',
  });
  highPrice.associate = function(models) {
    highPrice.belongsTo(models.position, {foreignKey: 'positionId'});
  };
  return highPrice;
};

File: backendSharp/Models/HighPrice.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;


namespace Trowoo.Models
{
    public class HighPrice
    {
        public int Id {get; set;}
        [Required]
        public decimal Price {get; set;}
        [Required]
        [ForeignKey("PositionId")]
        public int PositionId {get; set;}
    }
}